scripts/async_profiler_build_shared.sh

```diff
@@ -5,8 +5,8 @@
 #
 set -euo pipefail
 
-VERSION=v2.7g2
-GIT_REV="6541b573294c3c9e8efb633bc830440a6f2e13d0"
+VERSION=v2.8g2
```

Comment:
Do we want to use `v2.8g2` or `v2.8g4`? You posted the `range-diff` comparing g2 and g4

Reply:
`v2.8g4` but will probably wait for v2.8.1 because we have multiple fixes not included in the release yet